### ISSUE TITLE
Loading should be indicated while downloading "My Uploads"

### ISSUE THREAD

**Issue by nextLane** [commons-app/apps-android-commons#374]
Presently, on logging the app, I am shown MyUploads 0 and after some time it reflects all the uploads, when the pics are done downloading, a loading indicator while that time duration may improve user experience.

**Comment by misaochan** [comment]
Interesting, does it not say "Waiting for first sync" for you instead of just "0 uploads"?

Regardless, good idea, thanks - a loading indicator would be far better than just that phrase. :)

**Comment by nextLane** [comment]
@misaochan Yes, it does, but for lesser duration than it takes to download the pics. 
Thanks :)

**Comment by tobias47n9e** [comment]
Not sure if it is possible to embed in our current code, but animated SVG are quite nice for these things. Made an example here: https://commons.wikimedia.org/wiki/File:SVG_animated_loading_icon.svg

Maybe someone wants to come up with a few good loading icons and then we can decide which one to use.

**Comment by maskaravivek** [comment]
In my opinion putting an [animated SVG](https://developer.android.com/reference/android/graphics/drawable/AnimatedVectorDrawable.html) would require considerable effort. My uploads page could have a progress bar or dialog similar to the Nearby page.

**Comment by misaochan** [comment]
Yeah, I think a standard progress bar would be more than sufficient, but if someone really wants to have a go at submitting a patch for an animated SVG, I'm happy to merge unless there are other drawbacks to using one. :)

**Comment by Bluesir9** [comment]
I believe this has not been done as yet, in which case I would like to pick this up.

**Comment by nicolas-raoul** [comment]
@Bluesir9 Go ahead, thanks! :-)
